### DISCUSSION TITLE
Multiple layouts for a single route file

### DISCUSSION THREAD

**Discussion by Tushar-Gautam** [remix-run/remix#9651] [Q&A]
I want to use 2 layouts for my faq route, the layouts are _full and _bodyContainer and the route is `_full.faq-> route.tsx`.
Now i want to use _full as well as __bodyContainer with the route,tried various permutations but its not working.How to resolve this.

**Comment by denlahodnyi** [comment]
I also wondered how to do it and finished with such a workaround. It works, but I'm not sure whether it's acceptable in remix. I'm using routes as folders

|-- **_foo**
|----|-- **route.tsx**     <-- renders ```<Outlet />```
|-- **_foo._bar**
|----|-- **route.tsx**     <-- renders ```<Outlet />```
|-- **_foo._bar._index**
|----|-- **route.tsx**     <-- renders home page (/) wrapped in foo and bar layouts
|-- **_foo._bar.some-page**
|----|-- **route.tsx**     <-- renders another page (/other-page) wrapped in foo and bar layouts
